packages/backend/src/services/ServiceRepository.ts

```diff
@@ -857,6 +859,10 @@ export class ServiceRepository
         return this.getService('aiService');
     }
 
+    public getAiAgentService<TAiAgentServiceImpl>(): TAiAgentServiceImpl {
```

Comment:
we could keep it as consistent as `getScimService` and do 

```suggestion
    public getAiAgentService<AiAgentServiceImplT>(): TAiAgentServiceImplT {
```